quarkchain/core.py

```diff
@@ -167,6 +167,25 @@ def deserialize(self, bb):
         return [self.ser.deserialize(bb) for i in range(size)]
 
 
+class BigUintSerializer:
+    """Not infinity, but pretty big: 2^2048-1"""
+
+    def __init__(self):
+        self.ser = PrependedSizeBytesSerializer(1)
```

Comment:
not used

Reply:
lol yeah I probbaly should simply use it

Reply:
done